### PR TITLE
Updating list topic

### PR DESCRIPTION
**Pull request by YoannaIvanova**

### CODE REVIEW

File: en/components/list.md [hunk 1]
```diff
@@ -51,83 +65,46 @@ The List Item has two distinct areas: Primary Action with non-interactive conten
...
-| Text + Icons     | <img class="responsive-img" src="../images/list_item_secondary5.png" srcset="../images/list_item_secondary5@2x.png 2x" /> |                                                                                                                                       |
-| Toggle           | <img class="responsive-img" src="../images/list_item_secondary6.png" srcset="../images/list_item_secondary6@2x.png 2x" /> | Switch in a Secondary Action can not have a label, therefore, the Label Style is set to None and this setting should not be changed   |
-| Two-line Numbers | <img class="responsive-img" src="../images/list_item_secondary7.png" srcset="../images/list_item_secondary7@2x.png 2x" /> |                                                                                                                                       |
+| Checkbox         | <img class="responsive-img" src="../images/list_item_secondary2.png" srcset="../images/list_item_secondary2@2x.png 2x" /> | Checkbox in a Secondary Action can not have a label, therefore, the Label Style is set to ~No Symbol and this setting should not be changed |
```
Comment (StefanIvanov) [inline en/components/list.md:84]:
```suggestion
| Checkbox         | <img class="responsive-img" src="../images/list_item_secondary2.png" srcset="../images/list_item_secondary2@2x.png 2x" /> | Checkbox in a Secondary Action can not have a label, therefore, the before and after labels are set to ~No Symbol and this setting should not be changed |
```

File: en/components/list.md [hunk 2]
```diff
@@ -51,83 +65,46 @@ The List Item has two distinct areas: Primary Action with non-interactive conten
...
+| Text and Icon            | <img class="responsive-img" src="../images/list_item_secondary3.png" srcset="../images/list_item_secondary3@2x.png 2x" /> |                                                                                                                                       |
+| Toggle             | <img class="responsive-img" src="../images/list_item_secondary4.png" srcset="../images/list_item_secondary4@2x.png 2x" /> | Switch in a Secondary Action can not have a label, therefore, the Label Style is set to ~ No Symbol and this setting should not be changed                                                                                                                 |
```
Comment (StefanIvanov) [inline en/components/list.md:86]:
```suggestion
| Toggle             | <img class="responsive-img" src="../images/list_item_secondary4.png" srcset="../images/list_item_secondary4@2x.png 2x" /> | Switch in a Secondary Action can not have a label, therefore, the before and after labels are set to ~ No Symbol and this setting should not be changed                                                                                                                 |
```

File: en/components/list.md [hunk 3]
```diff
@@ -51,83 +65,46 @@ The List Item has two distinct areas: Primary Action with non-interactive conten
...
 ## Styling
 
 The List comes with styling flexibility through the overrides available for background color and the different elements used in the List Items, such as icons and text, as well as components like Avatar, Badge, Checkbox, Icon, Progress, Switch, etc. with their own styling capabilities.
 
 <img class="responsive-img" src="../images/list_styling.png" srcset="../images/list_styling@2x.png 2x" />
 
+## List Preset
+
+In addition, you can find a variety of Presets you can choose from for quick list creation.
+
```
Comment (StefanIvanov) [inline en/components/list.md:97]: Assuming that you want to tell about the list patterns let's remove this from here and @sdimchevski will describe all list presets available as patterns in that topic.

Comment (StefanIvanov) [inline en/components/list.md:97]:
```suggestion
```

**Review by StefanIvanov — CHANGES_REQUESTED** [review]

File: en/components/list.md [hunk 4]
```diff
@@ -139,7 +116,6 @@ Related topics:
 - [Progress](progress.md)
 - [Switch](switch.md)
 - [Lists Pattern](../patterns/lists.md)
-- [Data Binding](../codegen/data-binding.md)
   <div class="divider--half"></div>
```
Comment (StefanIvanov) [inline en/components/list.md:120]:
```suggestion
```

File: en/components/list-skeleton.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
+---
+title: Skeleton List - Design System Component
+_description: The Skeleton List is a representation of a List component that is shown while data is being loaded in the background to provide content for a list.
```
Comment (StefanIvanov) [inline en/components/list-skeleton.md:3]:
```suggestion
_description: The Skeleton List is a representation of a List component that is shown while data is being loaded in the background to provide content for its items.
```

File: en/components/list-skeleton.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+|                              |                                                                                                  |                                                                                                                                            |
+| ---------------------------- | ------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| Avatar + Label + Description | <img class="responsive-img" src="../images/list_skeleton_item_primary1.png" srcset="../images/list_skeleton_item_primary1@2x.png 2x" />     |                                                                                                                                            |
+| Checkbox + Label               | <img class="responsive-img" src="../images/list_skeleton_item_primary2.png" srcset="../images/list_skeleton_item_primary2@2x.png 2x" />   |  Checkbox in a Secondary Action can not have a label, therefore, the before and after labels are set to ~No Symbol and this setting should not be changed |
```
Comment (StefanIvanov) [inline en/components/list-skeleton.md:44]:
```suggestion
| Checkbox + Label               | <img class="responsive-img" src="../images/list_skeleton_item_primary2.png" srcset="../images/list_skeleton_item_primary2@2x.png 2x" />   |  Checkbox in a Primary Action can not have a label, therefore, the before and after labels are set to ~No Symbol and this setting should not be changed |
```

File: en/components/list-custom.md [hunk 7]
```diff
@@ -38,19 +38,19 @@ The Custom List Item has high-level styling flexibility similar to the normal Li
...
 
 <img class="responsive-img" src="../images/list_item_custom_layout0.png" srcset="../images/list_item_custom_layout0@2x.png 2x" />
 
-1.  We will reuse the Header from the Primary Action Group but resize it to fit in the right half only and update the strings for the Title and Subtitle Text. In the Secondary Action Group, we will add a Raised Button position to it near the bottom right corner, updating its text and resizing accordingly to fit the new value. Lastly, let's remove the default Secondary Action, as we will not be needing Icons for our target custom layout.
+1.  First, we have to remove the paragraph and footer. Then we will reuse the Header from the Primary Action Group but resize it to fit in the right half only and update the strings for the Title and Subtitle Text. In the Secondary Action Group, we will add a Raised Button position to it near the bottom right corner, updating its text and resizing accordingly to fit the new value. Lastly, let's remove the default Secondary Action, as we will not be needing Icons for our target custom layout.
```
Comment (StefanIvanov) [inline en/components/list-custom.md:45]:
```suggestion
1.  First, we have to remove the paragraph and footer. Then we will reuse the Header from the Primary Action Group but resize it to fit in the right half only and update the strings for the Title and Subtitle Text. In the Secondary Action Group, we will add a Raised Button, position it near the bottom right corner, updating its text and resize it accordingly to fit the new value. Lastly, let's remove the default Secondary Action, as we will not be needing Icons for our target custom layout.
```

File: en/components/list-custom.md [hunk 8]
```diff
@@ -38,19 +38,19 @@ The Custom List Item has high-level styling flexibility similar to the normal Li
...
 
     <img class="responsive-img" src="../images/list_item_custom_layout1.png" srcset="../images/list_item_custom_layout1@2x.png 2x" />
 
-2.  Next, we will insert a List/Custom/Blocks/Image Content in the Primary Action Group and add a Badge on top of it within the same group. Now, we have to size the image according to the Item size and select content for it. After modifying the Badge, as well by updating its Value Text and hiding its Border and Elevation, we should have something similar to this.
+2.  Next, we will insert a List/Custom/Blocks/Image Content in the Primary Action Group and add a Badge on top of it within the same group. Now, we have to size the image according to the Item size and select content for it. After modifying the Badge, as well as after updating its Value Text and hiding Elevation, we should have something similar to this.
```
Comment (StefanIvanov) [inline en/components/list-custom.md:49]:
```suggestion
2.  Next, we will insert a List/Custom/Blocks/Image Content in the Primary Action Group and add a Badge on top of it within the same group. Now, we have to size the image according to the Item size and select content for it. After modifying the Badge, updating its Value Text, and setting its Elevation to None, we should have something similar to this.
```

**Review by StefanIvanov — CHANGES_REQUESTED** [review]

**Review by StefanIvanov — APPROVED** [review]

**Review by sdimchevski — APPROVED** [review]

**Pull request merged**